task/thermal/src/bsp/gimlet_b.rs

```diff
@@ -173,359 +120,374 @@ impl Bsp {
+        true,
+    ),
+    InputChannel::new(
+        TemperatureSensor::new(
+            Device::Dimm,
+            devices::tse2004av_dimm_h1,
+            sensors::TSE2004AV_DIMM_H1_TEMPERATURE_SENSOR,
+        ),
+        DIMM_THERMALS,
+        POWER_STATE_A0 | POWER_STATE_A2,
+        true,
+    ),
+    // U.2 drives
+    InputChannel::new(
+        TemperatureSensor::new(
+            Device::U2,
+            devices::nvmebmc_u2_n0,
+            sensors::NVMEBMC_U2_N0_TEMPERATURE_SENSOR,
+        ),
+        U2_THERMALS,
+        POWER_STATE_A0,
+        true,
+    ),
+    InputChannel::new(
+        TemperatureSensor::new(
+            Device::U2,
+            devices::nvmebmc_u2_n1,
+            sensors::NVMEBMC_U2_N1_TEMPERATURE_SENSOR,
+        ),
+        U2_THERMALS,
+        POWER_STATE_A0,
+        true,
+    ),
+    InputChannel::new(
+        TemperatureSensor::new(
+            Device::U2,
+            devices::nvmebmc_u2_n2,
+            sensors::NVMEBMC_U2_N2_TEMPERATURE_SENSOR,
+        ),
+        U2_THERMALS,
+        POWER_STATE_A0,
+        true,
+    ),
+    InputChannel::new(
+        TemperatureSensor::new(
+            Device::U2,
+            devices::nvmebmc_u2_n3,
+            sensors::NVMEBMC_U2_N3_TEMPERATURE_SENSOR,
+        ),
+        U2_THERMALS,
+        POWER_STATE_A0,
+        true,
+    ),
+    InputChannel::new(
+        TemperatureSensor::new(
+            Device::U2,
+            devices::nvmebmc_u2_n4,
+            sensors::NVMEBMC_U2_N4_TEMPERATURE_SENSOR,
+        ),
+        U2_THERMALS,
+        POWER_STATE_A0,
+        true,
+    ),
+    InputChannel::new(
+        TemperatureSensor::new(
+            Device::U2,
+            devices::nvmebmc_u2_n5,
+            sensors::NVMEBMC_U2_N5_TEMPERATURE_SENSOR,
+        ),
+        U2_THERMALS,
+        POWER_STATE_A0,
+        true,
+    ),
+    InputChannel::new(
+        TemperatureSensor::new(
+            Device::U2,
+            devices::nvmebmc_u2_n6,
+            sensors::NVMEBMC_U2_N6_TEMPERATURE_SENSOR,
+        ),
+        U2_THERMALS,
+        POWER_STATE_A0,
+        true,
+    ),
+    InputChannel::new(
+        TemperatureSensor::new(
+            Device::U2,
+            devices::nvmebmc_u2_n7,
+            sensors::NVMEBMC_U2_N7_TEMPERATURE_SENSOR,
+        ),
+        U2_THERMALS,
+        POWER_STATE_A0,
+        true,
+    ),
+    InputChannel::new(
+        TemperatureSensor::new(
+            Device::U2,
+            devices::nvmebmc_u2_n8,
+            sensors::NVMEBMC_U2_N8_TEMPERATURE_SENSOR,
+        ),
+        U2_THERMALS,
+        POWER_STATE_A0,
+        true,
+    ),
+    InputChannel::new(
+        TemperatureSensor::new(
+            Device::U2,
+            devices::nvmebmc_u2_n9,
+            sensors::NVMEBMC_U2_N9_TEMPERATURE_SENSOR,
+        ),
+        U2_THERMALS,
+        POWER_STATE_A0,
+        true,
+    ),
+];
+
+const MISC_SENSORS: [TemperatureSensor; NUM_TEMPERATURE_SENSORS] = [
```

Comment:
Looks like the order of this array changed - does that matter?

Reply:
Good question – I don't think so, because we're tagging readings with `SensorId` when posting to the `sensors` task.